src/coord/src/catalog.rs

```diff
@@ -720,7 +720,19 @@ impl Catalog {
                 .set_catalog_content_version(catalog_content_version)?;
         }
 
-        let items = catalog.storage().load_items()?;
+        let (catalog, extra_events) = Self::open_from(catalog)?;
+        events.extend(extra_events.into_iter());
+
+        Ok((catalog, events))
+    }
+
+    // Takes a catalog which is not "loaded" and thus cannot yet resolve names
```

Comment:
Can we add more detail about what `"loaded"` means in this context? Maybe something like:

> Takes a catalog which only has items in its on-disk storage ("unloaded") and cannot yet resolve names, and returns a catalog loaded with those items.

Or maybe another function name encapsulates what this does more clearly?

Reply:
I like yours better!